components/elm/src/biogeophys/ActiveLayerMod.F90

```diff
@@ -56,27 +58,41 @@ subroutine alt_calc(num_soilc, filter_soilc, &
-    integer  :: dtime                           ! time step length in seconds
-    integer  :: k_frz                           ! index of first nonfrozen soil layer
-    logical  :: found_thawlayer                 ! used to break loop when first unfrozen layer reached
-    real(r8) :: t1, t2, z1, z2                  ! temporary variables
+    integer  :: c, j, fc, g                             ! counters
+    integer  :: alt_ind                                 ! index of base of active layer
+    integer  :: year                                    ! year (0, ...) for nstep+1
+    integer  :: mon                                     ! month (1, ..., 12) for nstep+1
+    integer  :: day                                     ! day of month (1, ..., 31) for nstep+1
+    integer  :: sec                                     ! seconds into current date for nstep+1
+    integer  :: dtime                                   ! time step length in seconds
+    integer  :: k_frz                                   ! index of first nonfrozen soil layer
+    logical  :: found_thawlayer                         ! used to break loop when first unfrozen layer reached
+    real(r8) :: t1, t2, z1, z2, orig_excess, old_mfrac  ! temporary variables
+    real(r8), dimension(nlevgrnd) :: melt_profile       ! profile of melted excess ice
     !-----------------------------------------------------------------------
 
-    associate(                                                                & 
-         t_soisno             =>    col_es%t_soisno        ,    & ! Input:   [real(r8) (:,:) ]  soil temperature (Kelvin)  (-nlevsno+1:nlevgrnd)                    
-         
-         alt                  =>    canopystate_vars%alt_col             ,    & ! Output:  [real(r8) (:)   ]  current depth of thaw                                                 
-         altmax               =>    canopystate_vars%altmax_col          ,    & ! Output:  [real(r8) (:)   ]  maximum annual depth of thaw                                          
-         altmax_lastyear      =>    canopystate_vars%altmax_lastyear_col ,    & ! Output:  [real(r8) (:)   ]  prior year maximum annual depth of thaw                               
-         alt_indx             =>    canopystate_vars%alt_indx_col        ,    & ! Output:  [integer  (:)   ]  current depth of thaw                                                  
-         altmax_indx          =>    canopystate_vars%altmax_indx_col     ,    & ! Output:  [integer  (:)   ]  maximum annual depth of thaw                                           
-         altmax_lastyear_indx =>    canopystate_vars%altmax_lastyear_indx_col & ! Output:  [integer  (:)   ]  prior year maximum annual depth of thaw                                
+    ! RF NOTE: use of 1989 ALT in these parameterizations is somewhat of a placeholder used to compare against
+    ! ATS runs for NGEE Arctic Phase 3. It should ultimately be replaced by a more physically based threshold
```

Comment:
It appears these are notes from code development and are not comments. They should be deleted or updated.

Reply:
Comments deleted